src/components/Navbar/Navbar.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { useState } from 'react';
+import './Navbar.css';
+import web from '../../assets/talawa-logo-lite-200x200.png';
```

Comment:
Please try to use a descriptive name for your asset imports, especially since Javascript allows you to name your asset imports

Since you are importing the Logo here, the import would be
```
import Logo from '../../assets/talawa-logo-lite-200x200.png';
```

And to clean up the imports further, we had set up absolute imports with the base directory being `src` so you could import this asset as

```
import Logo from 'assets/talawa-logo-lite-200x200.png';
```